app/assets/javascripts/dragdrop.js

```diff
@@ -32,20 +32,10 @@ jQuery(function() {
     $('#side-dropzone').removeClass('hover');
   });
   $('.dropzone').on('drop',function(e) {
-    // this 'drop' listener is also reused for pages with just one form, ie. /wiki/new
-    const closestCommentFormWrapper = e.target.closest('div.comment-form-wrapper'); // this returns null if there is no match
+    const params = getEditorParams(e.target); // defined in editorHelper.js
     e.preventDefault();
```

Comment:
Am I doing this right: calling on editorHelper.js? I browsed the Rails guide to JS assets, but am not sure if what I'm doing falls under 'best practices'. Would appreciate some pointers if you have any.

Reply:
Yes that looks fine to me